src/libcore/ops.rs

```diff
@@ -171,6 +179,44 @@ use marker::Unsize;
 ///     let _x = HasDrop;
 /// }
 /// ```
+///
+/// Showing the recursive nature of `Drop`. When `outer` goes out of scope, the
+/// `drop` method will be called for `Outer` and then the `drop` method for
+/// `Inner` will be called. Therefore `main` prints `Dropping Outer!` and then
```

Comment:
repetitive wording "the `drop` method will be called for `Outer` and then the `drop` method for `Inner` will be called."
Why not "the `drop` method will be called first for `Outer`, then for `Inner`"